add cattle-cluster-agent debug logs (#842)
rancher/rancher-docs: docs/troubleshooting/other-troubleshooting-tips/logging.md

Just like the `trace` log level above, `rancher-machine` debug logs can contain sensitive information.

:::

## Cattle-cluster-agent debug logs

> If you need to troubleshoot the initialization of `cattle-cluster-agent` the log levels can be set during the 
> initialization of the downstream clusters.  

Reviewer: ```suggestion
The `cattle-cluster-agent` log levels can be set when you initialize the downstream clusters.  
```

> Wile creating a Cluster under `Cluster Configuration -> Agent Environment Vars` some variables can be set to define
> the log level.  

Reviewer: ```suggestion
When you create a cluster under **Cluster Configuration > Agent Environment Vars** you can set variables to define the log level.  
```

> For trace-level `CATTLE_TRACE` or `RANCHER_TRACE` must be set to `true`
> 
> For debug-level `CATTLE_DEBUG` or `RANCHER_DEUBG` must be set to `true`

Reviewer: ```suggestion
- Trace-level logging: Set `CATTLE_TRACE` or `RANCHER_TRACE` to `true`

-  Debug-level logging: Set `CATTLE_DEBUG` or `RANCHER_DEUBG` to `true`
```

:::caution

> Just like the `trace` log level above, `cattle-cluster-agent` debug logs can contain sensitive information.

Reviewer: ```suggestion
The `cattle-cluster-agent` debug logs may contain sensitive information.
```
